Packages/KeyAppKit/Sources/KeyAppNetworking/HTTPClient/HTTPEndpoint.swift

```diff
@@ -1,16 +1,10 @@
-//
-//  Endpoint.swift
-//  p2p_wallet
-//
-//  Created by Ivan on 28.04.2022.
-//
-
 import Foundation
 
-public protocol Endpoint {
+/// Endpoint for specific `HTTPClient` network call
```

Comment:
Why do we need this protocol? Can we use URLRequest instances instead?

Reply:
Lets discuss in https://github.com/p2p-org/p2p-wallet-ios/pull/1249